src/Library/KataScript.hpp

```diff
@@ -3509,10 +3515,111 @@ namespace KataScript {
 
         prevState = tempState;
 	}
+	// The is used to check for 'import' statments
+	void KataScriptInterpreter::runPreCompiler(char* source) {
```

Comment:
This is unnecessarily complex, all we need to do is call the existing parse file function on the new source path